src/pages/gen2/reference/cli-commands/index.mdx

```diff
@@ -217,6 +217,90 @@ npx amplify generate config --stack amplify-nextamplifygen2-josef-sandbox-ca85e1
+  - `--app-id`(String) - The Amplify App ID of the project.
+  - `--branch`(String) - A git branch of the Amplify project.
+
+Optional parameters:
+- `--debug` (boolean) - Print debug logs to the console.
+- `--debug`(boolean) - Enables printing debug logs to the console.
+- `--help`(boolean) - Displays help information about the command.
+- `--profile`(string) - Specifies an AWS profile name.
+- `--out`(string) - Specifies the path to the directory where the config is written. If not provided, defaults to the current process working directory.
+- `--format`(string (choices: "modelgen", "graphql-codegen", "introspection")) - Specifies the format of the GraphQL client code to be generated.
+- `--model-target` (string (choices: "java", "swift", "javascript", "typescript", "dart")) - Specifies the modelgen export target. Only applies when the --format parameter is set to modelgen.
+- `--statement-target`(string (choices: "javascript", "graphql", "flow", "typescript", "angular")) - Specifies the graphql-codegen statement export target. Only applies when the --format parameter is set to graphql-codegen.
+- `--type-target`(string (choices: "json", "swift", "typescript", "flow", "scala", "flow-modern", "angular")) - Specifies the optional graphql-codegen type export target. Only applies when the --format parameter is set to graphql-codegen.
+- `--all`(boolean)- Shows hidden options.
+
+<SubCommandHeading parentCommand="amplify-graphql-client-code">Usage</SubCommandHeading>
+
+
+### Generate GraphQL client code using the Amplify App ID and branch. 
+```bash
+npx amplify generate graphql-client-code --app-id xxxxxx	--branch staging
+```
+### Generate GraphQL client code for a branch that is connected in the Amplify service
+Sometimes you want to test your latest local changes with the backend of another deployed branch. If you want to generate the GraphQL client code file(s) for the latest deployment of another branch, then you can run the following command:
+
+```bash 
+npx amplify generate graphql-client-code --branch staging
+```
+### Generate codegen for CDK app using a joint "AmplifyBackendStack" construct
+Assume you have deployed your Amplify project with the CDK construct. You will need to remember your app's project name (designated as the second parameter in your CDK construct) and stack name (designated as part of your `npx cdk deploy` context)
+
+// CDK code in stack.ts
+```ts
+import { Construct } from 'constructs';
+import { App, Backend } from 'aws-cdk-lib/aws-amplify';
+
+export class HelloCdkStack extends cdk.Stack {
+  constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    new Backend(this, "my-amplify-backend", { /* ... */ });
+  }
+}
+```
+// Deployment command for CDK project
+```bash
+npx cdk deploy
+```
+Run Amplify codegen command to generate GraphQL codegen:
+```bash
+npx amplify generate graphql-client-code --stack="my-amplify-stack" --platform ts --out ./src
```

Comment:
I can't find `my-amplify-stack` in the code snippets above. So it doesn't quite "connect" in my mind.

Reply:
updated the correct stack name